index.html

```diff
@@ -16,8 +16,7 @@ <h1 class="page-header-title animated fadeInDown"><span class="text-primary">Wom
                         <img src="/assets/images/logo.png" alt="logo">
                     </div>
                     <h5 class="sub-title animated fadeIn">The professional network for women who work in robotics - or
-                        who
-                        aspire to.</h5>
+                        who aspire to. Non-binary people welcome.</h5>
```

Comment:
What if we put this on its own line?

```suggestion
                        who aspire to.</h5>
                       <h5 class="sub-title animated fadeIn">Non-binary people welcome.</h5>
```